lib/rules/image-needs-alt.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ESLintUtils } from "@typescript-eslint/utils";
+import { makeLabeledControlRule } from "../util/ruleFactory";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const rule = ESLintUtils.RuleCreator.withoutDocs(
+    makeLabeledControlRule({
+        component: "Image",
+        messageId: "imageNeedsAlt",
+        description: "Accessibility: Image must have alt attribute with a meaningful description of the image",
```

Comment:
it'll be good to tell the users to use alt='' if the image is purely decorative, although it is rare